src/app/pages/data-protection/cloudsync/cloudsync-form/cloudsync-form.component.ts

```diff
@@ -151,6 +151,7 @@ export class CloudSyncFormComponent implements OnInit {
     { label: 'One Zone-IA', value: 'ONEZONE_IA' },
     { label: 'Intelligent-Tiering', value: 'INTELLIGENT_TIERING' },
     { label: 'Glacier', value: 'GLACIER' },
+    { label: 'Glacier Instant Retrieval', value: 'GLACIER_IR' },
```

Comment:
Selection of this option causes validation error from middleware.

`cloud_sync_create.attributes.storage_class: Invalid choice: GLACIER_IR [EINVAL]`

Looks like it's not supported by middleware yet.

Reply:
A PR #14094 has been submitted for middleware.  But it's not getting reviewed.  How do I get it reviewed?

Reply:
I've pinged some people to review.

Reply:
Middleware PR was merged.